pkg/signup/signup_service_test.go

```diff
@@ -450,6 +450,7 @@ func (s *TestSignupServiceSuite) TestGetSignupStatusOK() {
 	assert.True(s.T(), response.Status.Ready)
 	assert.Equal(s.T(), response.Status.Reason, "mur_ready_reason")
 	assert.Equal(s.T(), response.Status.Message, "mur_ready_message")
+	assert.False(s.T(), response.Status.VerificationRequired)
```

Comment:
shouldn't we check this param also in the other cases?

Reply:
There's already an explicit test for this property, see `TestUserWithExcludedDomainEmailSignsUp()` further up.  The prior test `TestCreateUserSignup()` tests it also.

Reply:
Ok, I missed that. But correct me if I'm wrong - none of those tests that you mentioned verifies the `GetSignup` method, right? In other words, they don't verify getting of the already existing `UserSignup` resource, but they verify creation of the new one.

Reply:
Good catch, you're absolutely right.  I've updated the tests that check the Signup status returned from `GetSignup` to also confirm that `verificationRequired` is correctly set.

Reply:
awesome, thanks :+1: